### DISCUSSION TITLE
levels per user

### DISCUSSION THREAD

**Discussion by Kyzz-z** [MrAugu/discord-xp#64] [Q&A] [answered]
can i make it so that when a user levels up it saves it to all guilds? so the levels dont change even when you're in a different guild. so for example, you level up to 2 in your friend's server with your bot. Now you go onto another server and check your level and its still 2. i dont wanna make the levels unique per guild, Hopefully i explained this clearly

**Comment by Myst82015** [comment, marked as answer]
Yes that is possible.
The package doesn't check the guild id you're providing. This means you can provide `anything` you want.
Example: instead of a regular guild id like `'16435468435465464465'` you provide `'global'`
-> now you can fetch the level when you use `'global`

**Reply by Kyzz-z** [reply]
so i do `Levels.fetch(message.author.id, global)`? but wouldnt that say that global is undefined?, please elaborate or maybe im just dumb`

**Reply by Myst82015** [reply]
The guild id has to be provided as a string. 
->`Levels.fetch(message.author.id, "global")`
